supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,11 +2,44 @@
 [[links]]
 = Links
 
-// [[internal-links]]
-// == Internal links
+[[cross-references]]
+== Cross-references
 
-// [[external-links]]
-// == External links
+Follow these guidelines when adding cross-references within your documentation:
+
+* Include cross-references only when necessary.
+* If the information is critical, consider including it instead of cross-referencing.
+
+.Example AsciiDoc: Cross-reference
+----
+For more information about <topic>, see xref:<link>[<link_text>].
+----
+
+[[external-links]]
+== External links
+
+Follow these guidelines when linking externally:
+
+* Avoid links to external sites when possible. External links can change or be unreliable.
+* Use hyperlinks unless the URL is an example URL or is otherwise inaccessible to users. See the _IBM Style Guide_ for information about using addresses in examples.
+* Default to followable, crawlable links. Do not use "nofollow" links unless absolutely necessary.
+* Do not use bare URLs. Bare URLs are unhelpful because they do not provide adequate context about the link target.
+* Avoid deep links (to a specific page or image from another company) to prevent an ongoing maintenance responsibility and to prevent bypassing a website's legal information.
+
+.Example AsciiDoc: External link
+----
+For more information about <topic>, see link:<link>[<link_text>].
+----
+
+[[link-text]]
+== Link text
+
+Follow these guidelines when specifying link text:
+
+* Contextually describe what the user will find at the target location so that they can decide if they want to leave their current location.
+* Use a concise sentence or sentence fragment as the link text.
+* Do not use bare URLs. Bare URLs are unhelpful because they do not provide adequate context about the link target.
```

Comment:
Perhaps we should define what is meant by "bare URL". I assumed that a bare URL is a link without link text, so I'm not sure if this point makes sense in this section, given that it's already included in the "External links" section? Perhaps it should be included in the Cross-references section instead?

Reply:
@bredamc I actually thought about changing "bare URL" to "raw URL" because that's the term I'm more familiar with. wdyt? Or I could give an example if that would help.

I do think that the raw URL piece is more applicable in the external links section, since I don't even think it's possible to give a raw URL for cross-references.

Reply:
@bergerhoffer Apologies for not being more clear -- we should absolutely retain that point in the "External links" section. I meant that we should include it in the "Cross-references" section instead of in the "Link text" section, but I agree that there's no need if it's not possible to provide a raw URL in a cross-reference anyway! 

Whether we call it a bare URL or a raw URL, I think we should define what it is. When we say "Do not use bare URLs", do we mean "Always include link text"? If so, should we phrase it in that way for clarity? If we retain this point in the "Link text" section, should that text be moved to be the first point?

Reply:
@bergerhoffer Apologies for the delay.

I think we should rephrase the current "Bare URLs" text as follows:
"Always include link text. _Bare URLs_ (URLs without explanatory information) are unhelpful because they do not provide adequate context about the link target."

Reply:
Examples are always good -- similar to the picture that paints a thousand words ;)

Reply:
I think your suggested edit makes it crystal clear, thanks! Updated

Reply:
One last tiny comment -- maybe reorder the list of points to list the "Do" statements first, and then the "Don't" statements?

Reply:
Done!